### PR TITLE
Fix optional time_limit access in create of question sets #11

### PR DESCRIPTION
**Pull request by tjchambers**
Optional time_limit value was not provided access by PHP.
Add test to verify access to time_limit.
Add test to confirm expired response.
Bump to v4.0.3

### CODE REVIEW

**Comment by tjchambers** [comment]
@alain ready for review and packaging when green

File: tests/QuestionSetTest.php [hunk 1]
```diff
@@ -35,6 +35,24 @@ public function testCreateQuestionSet()
...
+        $qs = $person->question_sets->create(17);
+
+        $this->assertSame($qs->time_limit, 17);
+    }
+
+    public function testTimeLimitExpired()
+    {
+        $person = self::createTestPerson();
+        $qs = $person->question_sets->create(2);
+        sleep(4);
```
Comment (alainmeier) [inline tests/QuestionSetTest.php]: This sleep is necessary right?

Comment (tjchambers) [inline tests/QuestionSetTest.php]: Yes - we are proving that a short time limit expires and the client receives the expired flag properly set as NOW expired. I made it short, but it still adds briefly to the test run. But it confirms that it is returned AND it is accessible to the client code.

**Comment by alainmeier** [comment]
As long as the sleep is required, this LGTM

**Pull request merged**